wikipedia_transform/main.py

```diff
@@ -0,0 +1,15 @@
+from wikipedia_transform import WikipediaTransform
+from wikipedia_transform.models import settings
+from wikipedia_transform.models.types import RecordType
+
+
+def main() -> None:
+    """
+    Invoke WikipediaTransform.transform() to transform Wikipedia data files into embeddings.
+    """
+    if settings.record_type == RecordType.WIKIPEDIA:
```

Comment:
This should really be a command line parameter and not a setting. Settings are for things like OpenAI keys.

Check out the Typer library.